frontend/src/services/integrationAppService.ts

```diff
@@ -0,0 +1,31 @@
+import axios from '~/utilities/axios';
+
+type IntegrationAppStatus = {
+  isInstalled: boolean;
+  isEnabled: boolean;
+  canInstall: boolean;
+  error: string;
+};
```

Comment:
We have the type stored in `frontent/src/types.ts` -- lets use that and not have a dupe.

Reply:
Done.